packages/vscode-ui5-language-assistant-bas-ext/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@ui5-language-assistant/vscode-ui5-language-assistant-bas-ext",
+  "displayName": "UI5 Language Assistant",
+  "description": "Language Support For SAPUI5",
```

Comment:
this is the `description` of the vsix inside of this wrapper.
not the description of this wrapper...